coalib/parsing/CliParsing.py

```diff
@@ -87,3 +88,27 @@ def parse_custom_settings(sections,
                                origin=origin,
                                section_name=key_touple[0],
                                from_cli=True)
+
+
+def check_conflicts(sections):
```

Comment:
No, it currently doesn't support what we need: `[-a | -b] [-a | -c]`. It only supports mutually exclusive arguments like this: `[-a | -b | -c]`